pocs/tests/test_sensor_recorder.py

```diff
@@ -0,0 +1,113 @@
+# Test sensors.py ability to read from two sensor boards.
+
+import collections
+import time
+
+from pocs.sensors import sensor_recorder
+
+
+def make_reading(name, v):
```

Comment:
For clarity's sake, `v` is a little lacking, especially since it is used as a timestamp and as a a field value.

Reply:
Done.

Reply:
I'm not seeing this change.